src/app/app.js

```diff
@@ -0,0 +1,44 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var Fluxible = require('fluxible');
+var routrPlugin = require('fluxible-plugin-routr');
+var pageRegistry = require('./page-registry');
+
+/**
+ * Create fluxible app.
+ */
+
+var app = new Fluxible({
+  appComponent: React.createFactory(require('./app-component'))
```

Comment:
I'd put all requires at the top. No need for an inline require, except in the unusual case of dynamically including things in the server.